tabulka.html

```diff
@@ -4,13 +4,15 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>data</title>
+    <title>Data </title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
 <h1>Data</h1>
 
 <!-- Table with Header, Footer, and Caption -->
-<table id="data" border="1">
+<table id="data">
```

Comment:
jen pro uplnost, psal jsem to pak i na discordu, tohle byla chyba zadani a radky se pridavaji omylem do tfoot elementu. Oprava by byla dat ten id atribut do elementu tbody, misto table.